cmd/geth/usage.go

```diff
@@ -103,6 +103,9 @@ var AppHelpFlagGroups = []flagGroup{
 		Flags: []cli.Flag{
 			utils.DeveloperFlag,
 			utils.DeveloperPeriodFlag,
+			utils.KeyStoreDirFlag,
+			utils.PasswordFileFlag,
+			utils.MinerEtherbaseFlag,
```

Comment:
Please remove these flags. They are already declared elsewhere, no point in duplicating it.

Reply:
Done with https://github.com/ethereum/go-ethereum/pull/21301/commits/e5c5333a4c319f88751d4495857d0e5ac758aebd